.travis.yml

```diff
@@ -1,5 +1,10 @@
 language: c
 
+sudo: required
```

Comment:
This was added since the integration tests run in a docker container and this requires sudo to run a docker container.